learn/tooling-guide/ballerina-shell.md

```diff
@@ -2,11 +2,197 @@
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Ballerina Shell
 permalink: /learn/tooling-guide/ballerina-shell/
+description: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 active: ballerina-shell
-intro: The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger. Thus, you can run or debug your Ballerina programs easily via the VS Code Ballerina extension by launching its debugger. 
+intro: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
 redirect_from:
   - /learn/tooling-guide/ballerina-shell
 ---
 
+It allows you to evaluate snippets of code without having to write complete programs. 
 
+This will particularly be your interest if you are looking to try out the language and its new features, and also for debugging purposes.
```

Comment:
"This will particularly be your interest" doesn't sound right here. Why change the original phrasing?

Reply:
The best practise in docs is to address the user directly and not in the indirect form.

Reply:
I see. Then how about phrasing it like this? 
"This will particularly be useful if you are looking to try out the language..."

Reply:
+1 Updated.